styles.css

```diff
@@ -0,0 +1,72 @@
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Inter', sans-serif;
+}
+
+body {
```

Comment:
- [ ] On this project we have to follow the Figma design, you can see on the screenshots below, that on the template the background "bleeds" through the padding, kindly change it so it better follows the design, changing the padding from the body to the header and the section tag class, helps achieve that effect, feel free to tweak the background-size from cover if you feel the need to
Figma:
![image](https://user-images.githubusercontent.com/73140461/143513427-d6096bae-48ca-4f3e-9da0-21caca785de4.png)
Yours: 
![image](https://user-images.githubusercontent.com/73140461/143513456-f6316af9-4985-4806-8232-dd704f60d9a6.png)